internal/services/machinelearning/machine_learning_workspace_resource.go

```diff
@@ -108,16 +108,18 @@ func resourceMachineLearningWorkspace() *pluginsdk.Resource {
 			},
 
 			"public_access_behind_virtual_network_enabled": {
```

Comment:
if this is going to be removed in v4.0 of the provider can we please wrap it in a 4.0 flag so its removed when the flag is true?

Reply:
updated